run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 [arch] [debug]"
+fi
+
+if [ $# -eq 2 ] && [ $2 == "debug" ]; then
+    echo "Enable GDB debug mode"
+    DBG="-s -S"
+fi
+
+case $1 in
+    x86_64)
+        qemu-system-x86_64 -kernel arch/x86/boot/bzImage \
+                           -append "rdinit=/linuxrc console=ttyS0" -nographic \
+                           --virtfs local,id=kmod_dev,path=$PWD/kmodules,security_model=none,mount_tag=kmod_mount \
+                           -enable-kvm \
+                           $DBG ;;
```

Comment:
some error when i run your patch:

figo@figo-OptiPlex-9020:~/work/test1/runninglinuxkernel/runninglinuxkernel_4.0$ sh run.sh x86_64
qemu-system-x86_64: --virtfs local,id=kmod_dev,path=/home/figo/work/test1/runninglinuxkernel/runninglinuxkernel_4.0/kmodules,security_model=none,mount_tag=kmod_mount: 9pfs Failed to initialize fs-driver with id:kmod_dev and export path:/home/figo/work/test1/runninglinuxkernel/runninglinuxkernel_4.0/kmodules

Reply:
Due to it is my first github commit. So some mistake caused by move the commit to fork's one.
just ignore that. I will create a new pull request for that.